ui/app/AppMain.qml

```diff
@@ -245,18 +248,14 @@ RowLayout {
                 visible: !profileModel.mnemonic.isBackedUp && sLayout.children[sLayout.currentIndex] !== profileLayoutContainer
                 anchors.top: parent.top
                 anchors.left: parent.right
-                anchors.leftMargin: -10
-                anchors.topMargin: -5
-                radius: width / 2
+                anchors.leftMargin: -17
```

Comment:
Same here. It's better to use `anchors.right`

Reply:
In here, you are right :)